### DISCUSSION TITLE
Spoiler markdown is missing arrow in Safari

### DISCUSSION THREAD

**Discussion by timagixe** [community/community#16079] [Projects and Issues]
Hi!

<details>

<summary>
SPOILER
</summary>

There is now `arrow icon` in front of `SPOILER` text in Safari browser, while in Chrome its present.

</details>

Seems like issues Is only observed for /issues and /gists. Links below:
- [Gist](https://gist.github.com/jbsulli/03df3cdce94ee97937ebda0ffef28287)
- [Issue](https://github.com/codesandbox/codesandbox-client/issues/6673)

**Screenshots:**
- **Chrome**
<img width="197" alt="Screen Shot 2022-05-05 at 9 46 09 AM" src="https://user-images.githubusercontent.com/60817786/166874451-78328d8b-b4cc-466a-82fc-6ef9a55ba129.png">

<img width="153" alt="Screen Shot 2022-05-05 at 9 46 14 AM" src="https://user-images.githubusercontent.com/60817786/166874455-03c527a0-3326-4465-a931-b06f493ce429.png">

------

- **Safari**
<img width="184" alt="Screen Shot 2022-05-05 at 9 45 50 AM" src="https://user-images.githubusercontent.com/60817786/166874483-3dedfde4-d675-404d-bfab-e6afa7049686.png">

<img width="168" alt="Screen Shot 2022-05-05 at 9 45 56 AM" src="https://user-images.githubusercontent.com/60817786/166874485-f218d397-db23-4129-bb5c-c5832436a0ed.png">

------

**Environment:**
Safari - Version 14.1.2 (14611.3.10.1.7)
Chrome - Version 100.0.4896.127 (Official Build) (x86_64)

**Comment by msagarpatel** [comment]
On Safari 15.4, I cannot see the arrow on pull requests either, but I do see it in this discussion.

I haven't tested Chrome, but this bug is not present on Firefox 99.0.1 (64-bit) on macOS 12.3.

**Reply by christos** [reply]
Same thing is happening on Safari 15.5 (17613.2.7.1.8) and macOS 12.4

**Reply by msagarpatel** [reply]
This appears to be fixed.
